review/main.tf

```diff
@@ -43,28 +43,110 @@ resource "google_compute_subnetwork" "subnet" {
   region        = "europe-west2"
 }
 
-module "davids_review_notebook" {
+resource "google_project_iam_custom_role" "notebook_user_role" {
+  role_id     = "notebookUserRole"
+  title       = "notebook-user-role"
+  description = "Role to allow AI Notebook user access"
+  permissions = [
+    "iam.serviceAccounts.actAs",
+    "compute.acceleratorTypes.list",
+    "compute.diskTypes.list",
+    "compute.instances.list",
+    "compute.machineTypes.list",
+    "compute.subnetworks.list",
+    "compute.instances.start",
+    "compute.instances.stop",
+    "notebooks.environments.get",
+    "notebooks.environments.getIamPolicy",
+    "notebooks.environments.list",
+    "notebooks.instances.get",
+    "notebooks.instances.getIamPolicy",
+    "notebooks.instances.list",
+    "notebooks.locations.get",
+    "notebooks.locations.list",
+    "notebooks.operations.get",
+    "notebooks.operations.list",
+    "notebooks.instances.update",
+    "notebooks.instances.start",
+    "notebooks.instances.stop"
+  ]
+}
+
+resource "google_service_account" "service_account_1" {
+  account_id   = "service-account-1"
+  display_name = "service-account-1"
+}
+
+resource "google_service_account" "service_account_2" {
+  account_id   = "service-account-2"
+  display_name = "service-account-2"
+}
+
+//resource "google_service_account_iam_member" "service_account_1_binding" {
+//  service_account_id = google_service_account.service_account_1.name
+//  role               = "roles/iam.serviceAccountUser"
+//  member             = "user:satya.bingumalla@ext.ons.gov.uk"
+//}
+
+resource "google_service_account_iam_member" "service_account_2_binding" {
+  service_account_id = google_service_account.service_account_2.name
+  role               = "roles/iam.serviceAccountUser"
+  member             = "user:"
```

Comment:
Not sure I'm following this bit

Reply:
Ah, this is just an example file :+1:

Reply:
Amended to line **94** in __To Review__ instructions above ☝️